packages/cli/templates/app/app/pages/index.tsx

```diff
@@ -115,15 +180,18 @@ const Home = () => (
     `}</style>
 
     <style jsx global>{`
+      @import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@300;700&display=swap');
+
       html,
       body {
         padding: 0;
         margin: 0;
-        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans,
-          Droid Sans, Helvetica Neue, sans-serif;
+        font-family: 'Libre Franklin', sans-serif;
```

Comment:
Add back default fallbacks for people who are doing this offline

Reply:
Yeah, keep what was there but add 'Libre Franklin' to the beginning of the list for folks who have it installed.

Reply:
fixed in my branch